src/YDialogSpy.cc

```diff
@@ -418,9 +419,8 @@ void YDialogSpyPrivate::refreshProperties()
 
     auto widget = selectedWidget();
     if (!widget) return;
-
+    YItemCollection items;
 	auto propSet = widget->propertySet();
-	YItemCollection items;
```

Comment:
what is goal of this change?

Reply:
This is unintended changes, thanks for spotting.